cmd/tofu/experiments.go

```diff
@@ -22,6 +22,6 @@ package main
 // open experiment.)
 var experimentsAllowed string
 
-func ExperimentsAllowed() bool {
```

Comment:
In having the function name match the variable name, we have better insight into the code when searching.  With the shortened name, using automation to find 'expAllowed' will miss instances of 'experimentsAllowed'.

It's a small detail, but I think it keeps the code base more understandable.

Consider retaining the longer function name.

Reply:
I intentionally made it different to reduce confusion. Neither is officially exported, though the variable is getting touched by compile-time `ldflags` games, so it made sense to me to keep them distinct.

Reply:
I don't think we can easily keep the same name, cause Go doesn't allow that.

But maybe experimentsAreAllowed or something like that for the function? Maybe experimentalFeaturesAllowed?

Having it exported to avoid the collision makes sense to me too.

I agree that expAllowed isn't obvious as to what it means.

Reply:
@cube2222 I went with `experimentsAreAllowed()`.